pkgs/applications/audio/netease-cloud-music/default.nix

```diff
@@ -0,0 +1,156 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, autoPatchelfHook
+, makeWrapper
+# , qt5
+, xorg
+, libdrm
+, alsa-lib
+, taglib
+, vlc
+, krb5
+, gtk2
+, gtk3
+, tcp_wrappers
+, e2fsprogs
+, gnutls
+, avahi
+, sqlite
+, libogg
+, libvorbis
+, libsndfile
+, libasyncns
+, dbus-glib
+, cups
+, nspr
+, gnome2
+, nss
+, icu60
+, libinput
+, mtdev
+, libpulseaudio
+  # , double-conversion
+  # , libjpeg_original
+  # , nettle
+  # , flac
+  # , eudev
+}:
+stdenv.mkDerivation rec {
+  pname = "netease-cloud-music";
+  version = "1.2.1";
+  src = fetchurl {
+    url = "https://d1.music.126.net/dmusic/netease-cloud-music_1.2.1_amd64_ubuntu_20190428.deb";
+    sha256 = "sha256-HunwKELmwsjHnEiy6TIHT5whOo60I45eY/IEOFYv7Ls=";
+    curlOpts = "-AMozilla/5.0";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+    makeWrapper
+  ];
+  # some libraries' versions don't match, comment them
+  buildInputs = [
+    stdenv.cc.cc.lib
+    # qt5.qtbase
+    # qt5.qtdeclarative
+    # qt5.qtwebchannel
+    # qt5.qtx11extras
+    # xorg.libXtst
+    xorg.libXScrnSaver
+    xorg.xcbutilkeysyms
+    xorg.xcbutilimage
+    xorg.xcbutilwm
+    xorg.xcbutilrenderutil
+    libdrm
+    alsa-lib
+    taglib
+    vlc
+    krb5
+    gtk2
+    gtk3
+    tcp_wrappers
+    e2fsprogs
+    gnutls
+    avahi
+    sqlite
+    libogg
+    libvorbis
+    libsndfile
+    libasyncns
+    dbus-glib
+    cups
+    nspr
+    gnome2.GConf
```

Comment:
GConf is deprecated and we are going to remove it.

Reply:
What is the alternative?